website/docs/d/list_latest_scans.html.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_list_latest_scans"
+description: |-
+  Get information about list_latest_scans
+subcategory: "Posture Management"
```

Comment:
Please use subcategory as Security and Compliance Center (SCC)

example - https://github.com/IBM-Cloud/terraform-provider-ibm/pull/3043/files#diff-d7a9dc26502531b3cbb5781353d133fc00bf67e3deca341b2fb5225ccd58ebeeR3